js/tweek-local-cache/src/split-join.ts

```diff
@@ -0,0 +1,25 @@
+export interface SplitJoin {
+  split(key: string): string[];
+  join(fragments: string[]): string;
+}
+
+export class MemoizedTweekKeySplitJoin implements SplitJoin {
+  private readonly _cache = new Map<string, string[]>();
+
+  split(key: string): string[] {
+    const cached = this._cache.get(key);
```

Comment:
Is it faster? I would assume hashing and simple string manipulation should have similar performance